Sources/BraintreeCore/ConfigurationLoader.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+class ConfigurationLoader {
+    
+    // MARK: - Private Properties
+    
+    private let configPath = "v1/configuration"
+    private let configurationCache: ConfigurationCache = ConfigurationCache.shared
+    private let http: BTHTTP
+    
+    init(http: BTHTTP) {
+        self.http = http
+    }
+    
+    deinit {
+        http.session.finishTasksAndInvalidate()
+    }
+    
+    func getConfig(_ authorization: ClientAuthorization, completion: @escaping (BTConfiguration?, Error?) -> Void) {
```

Comment:
Just a few more MARKs we usually use, but open to if folks don't find these useful

```suggestion
    // MARK: - Intitializer

    init(http: BTHTTP) {
        self.http = http
    }
    
    deinit {
        http.session.finishTasksAndInvalidate()
    }
    
    // MARK: - Internal Methods
    
    func getConfig(_ authorization: ClientAuthorization, completion: @escaping (BTConfiguration?, Error?) -> Void) {
```

Reply:
I like them, added: [418b39d](https://github.com/braintree/braintree_ios/pull/1353/commits/418b39d4f9c799335b6393f2fd9195fa034a674f)